### ISSUE TITLE
[BUG] Text only app indicators are not shown correctly.

### ISSUE THREAD

**Issue by prateekmedia** [cutefishos/statusbar#13]
There is no support for widgets in cutefish like KDE, so I decided to try a net speed app indicator, but due to cutefish only rendering iconitem of that, the indicator is just transparent and I can see the info only after hovering on it in a tooltip which is a bad behaviour.

Please fix this issue.

**Comment by RedBearAK** [comment]
Yes, AppIndicators are an important part of a functional desktop environment. Whether macOS, KDE Plasma or Ubuntu/GNOME, I always have around a dozen different indicators in the top bar for things like VPN clients, cloud storage services, clipboard managers, download managers, remote access tools and other utilities. 

There is a package this is missing that I usually need to get AppIndicator3 support in one of the applications that I use: 

```
gir1.2-appindicator3-0.1
```

For some reason this doesn't seem to exist in the Debian bullseye repo, but I was able to download it from the buster repo and it works fine.

**Comment by prateekmedia** [comment]
@RedBearAK What? I am not talking about app Indicator support, I know cutefish has that by installing that package.

But I wanted to know about the support of text only indicators like https://github.com/fossfreedom/indicator-sysmonitor

**Comment by RedBearAK** [comment]
@prateekmedia 

I see. I misunderstood. I do use some of those at times also. Thanks for submitting the issue.
